applications/nrf5340_audio/src/amod_main.c

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright(c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include "amod_api.h"
+#include "aobj_api.h"
+#include "data_fifo.h"
+#include "lc3_decoder.h"
+#include <ctype.h>
+#include <nrfx_clock.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <zephyr/kernel.h>
+#include <zephyr/shell/shell.h>
+
+/* Headers below only for testing */
+#include "amod_api_private.h"
+#include "lc3_decoder_private.h"
+
+#define MODULE_DATA_SIZE (4096)
+#define LC3_DECODER_DATA_SIZE AMOD_DATA_OUT_SET_SIZE(FRAME_SIZE_BYTES)
+#define LC3_DECODER_IN_MESSAGE_NUM (2)
```

Comment:
Nitpick: Suggest general to specific naming
```suggestion
#define LC3_DECODER_MESSAGE_IN_NUM (2)
```